# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gettext/hotfix-vasnprintf.patch.uclibc
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

vasnprintf.c: In function 'thousands_separator_wchar':
vasnprintf.c:502:44: error: '_NL_NUMERIC_THOUSANDS_SEP_WC' undeclared (first use in this function)
  502 |     (wchar_t) (unsigned long) nl_langinfo (_NL_NUMERIC_THOUSANDS_SEP_WC);

--- gettext-0.25.1/libtextstyle/lib/vasnprintf.c.vanilla	2025-07-18 20:33:31.487231808 +0200
+++ gettext-0.25.1/libtextstyle/lib/vasnprintf.c	2025-07-18 20:34:32.112300983 +0200
@@ -495,7 +495,7 @@
 static const wchar_t *
 thousands_separator_wchar (wchar_t stackbuf[10])
 {
-#  if __GLIBC__ >= 2 || defined __CYGWIN__
+#  if 0
   /* On glibc, in the unibyte locale fr_FR, the *wprintf routines use U+202F
      as separator, which cannot be represented in the locale encoding.  */
   stackbuf[0] =
--- gettext-0.25.1/gettext-runtime/intl/gnulib-lib/vasnprintf.c.vanilla	2025-07-18 20:35:21.313345423 +0200
+++ gettext-0.25.1/gettext-runtime/intl/gnulib-lib/vasnprintf.c	2025-07-18 20:35:30.457202957 +0200
@@ -495,7 +495,7 @@
 static const wchar_t *
 thousands_separator_wchar (wchar_t stackbuf[10])
 {
-#  if __GLIBC__ >= 2 || defined __CYGWIN__
+#  if 0
   /* On glibc, in the unibyte locale fr_FR, the *wprintf routines use U+202F
      as separator, which cannot be represented in the locale encoding.  */
   stackbuf[0] =
